src/LiveObjects.cpp

```diff
@@ -300,16 +299,15 @@ void LiveObjectsBase::sendData(const String customPayload) {
 }
 
 void LiveObjectsBase::publishMessage(const String& topic, JsonDocument& payload) {
-  outputDebug("Publishing message on topic");
-  outputDebug(topic);
+  outputDebug(INFO,"Publishing message on topic", topic);

```

Comment:
There is a missing space after "topic", leading to something like
_[INFO] Publishing message on **topicdev/data**_